gramex/services/__init__.py

```diff
@@ -902,3 +904,38 @@ def test(conf):
     # Remove auth: section when running gramex.
     # If there are passwords here, they will not be loaded in memory
     conf.pop('auth', None)
+
+
+def gramexlog(conf):
+    '''
+    '''
+    from elasticsearch import Elasticsearch
+    from elasticsearch import helpers
+    default_conf = conf.get('default')
```

Comment:
This only works for the `default:` app. It won't work for other servers. You'll need one connection for each server.